stdlib/testing/kapacitor/noop_yield.flux

```diff
@@ -0,0 +1,58 @@
+package testdata_test
+
+import "testing"
+
+option now = () => (2030-01-01T00:00:00Z)
+
+inData = "
+#datatype,string,long,dateTime:RFC3339,double,string,string
+#group,false,false,false,false,true,true
+#default,_result,,,,,
+,result,table,_time,_value,_measurement,user
+,,0,2018-05-22T19:53:26Z,0,CPU,user1
+,,0,2018-05-22T19:53:36Z,1,CPU,user1
+,,1,2018-05-22T19:53:26Z,4,CPU,user2
+,,1,2018-05-22T19:53:36Z,20,CPU,user2
+,,1,2018-05-22T19:53:46Z,7,CPU,user2
+,,1,2018-05-22T19:53:56Z,10,CPU,user2
+,,2,2018-05-22T19:53:26Z,1,RAM,user1
+,,2,2018-05-22T19:53:36Z,2,RAM,user1
+,,2,2018-05-22T19:53:46Z,3,RAM,user1
+,,2,2018-05-22T19:53:56Z,5,RAM,user1
+,,3,2018-05-22T19:53:26Z,2,RAM,user2
+,,3,2018-05-22T19:53:36Z,4,RAM,user2
+,,3,2018-05-22T19:53:46Z,4,RAM,user2
+,,3,2018-05-22T19:53:56Z,0,RAM,user2
+,,3,2018-05-22T19:54:06Z,2,RAM,user2
+,,3,2018-05-22T19:54:16Z,10,RAM,user2
+"
+
+outData = "
+#datatype,string,long,dateTime:RFC3339,double,string,string
+#group,false,false,false,false,true,true
+#default,1,,,,,
+,result,table,_time,_value,_measurement,user
+,,0,2018-05-22T19:53:26Z,0,CPU,user1
```

Comment:
our e2e doesn't check the `result` column but theoretically `yield(name:"1")`  would put `1` all the way down the second column in the CSV under `result`.